ext/http/http_next.rs

```diff
@@ -829,9 +849,48 @@ fn serve_http2_unconditional(
   io: impl HttpServeStream,
   svc: impl HttpService<Incoming, ResBody = HttpRecordResponse> + 'static,
   cancel: Rc<CancelHandle>,
+  h2_config: Http2Config,
```

Comment:
Instead of a defined struct, how about we accept a callback that can act on the builder directly providing more flexibility?

Reply:
that's a good idea! I'll fix

Reply:
Fixed [972ad96](https://github.com/denoland/deno/pull/26785/commits/972ad9679e316d94a21d85ac95be1e898ab4f93c)